mp/src/game/server/momentum/mom_replay_entity.cpp

```diff
@@ -70,12 +72,6 @@ void CMomentumReplayGhostEntity::Spawn()
     if (pPlayer)
     {
         SetGhostAppearance(pPlayer->m_playerAppearanceProps);
-        // now that we've set our appearance, the ghost should be visible again.
-        SetRenderMode(kRenderTransColor);
-        if (m_ghostAppearance.m_bGhostTrailEnable)
-        {
-            CreateTrail();
-        }
```

Comment:
If this is removed, where does the ghost's trail get created now?

Reply:
Ghost's trail is created after updating trail properties inside the base [class](https://github.com/momentum-mod/game/blob/2e2d5cd387fa5b5eb98e855b924fe34f1f728a87/mp/src/game/server/momentum/mom_ghost_base.cpp#L141)

Reply:
Ah okay. We may want to move the check for if it's enabled to inside CreateTrail so it doesn't create the trail if the appearance says not to.